migration.js

```diff
@@ -0,0 +1,125 @@
+require("dotenv").config();
```

Comment:
I would rename the`migration.js` file name to `trees_migration_milestone1.js`. There will be more migration scripts in this project so we need to differentitate each one.